spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/repository/JdbcDataflowTaskExecutionDao.java

```diff
@@ -0,0 +1,131 @@
+			+ "EXTERNAL_EXECUTION_ID, PARENT_EXECUTION_ID ";
+
+	/**
+	 * FROM clause for task execution.
+	 */
+	public static final String FROM_CLAUSE = "%PREFIX%EXECUTION";
+
+	/**
+	 * WHERE clause for running task.
+	 */
+	public static final String RUNNING_TASK_WHERE_CLAUSE = "where TASK_NAME = :taskName AND END_TIME IS NULL ";
+
+	/**
+	 * WHERE clause for task name.
+	 */
+	public static final String TASK_NAME_WHERE_CLAUSE = "where TASK_NAME = :taskName ";
+
+	private final NamedParameterJdbcTemplate jdbcTemplate;
+
+	private String tablePrefix = TaskProperties.DEFAULT_TABLE_PREFIX;
+
+	/**
+	 * Initializes the JdbcTaskExecutionDao.
+	 * @param dataSource used by the dao to execute queries and update the tables.
+	 * @param tablePrefix the table prefix to use for this dao.
+	 */
+	public JdbcDataflowTaskExecutionDao(DataSource dataSource, String tablePrefix) {
+		this(dataSource);
+		Assert.hasText(tablePrefix, "tablePrefix must not be null nor empty");
+		this.tablePrefix = tablePrefix;
+	}
+
+	/**
+	 * Initializes the JdbTaskExecutionDao and defaults the table prefix to
+	 * {@link TaskProperties#DEFAULT_TABLE_PREFIX}.
+	 * @param dataSource used by the dao to execute queries and update the tables.
+	 */
+	public JdbcDataflowTaskExecutionDao(DataSource dataSource) {
+		Assert.notNull(dataSource, "The dataSource must not be null.");
+		this.jdbcTemplate = new NamedParameterJdbcTemplate(dataSource);
+	}
+
+	private static final String DELETE_TASK_EXECUTIONS = "DELETE FROM %PREFIX%EXECUTION "
+			+ "WHERE task_execution_id in (:taskExecutionIds)";
+
+	private static final String DELETE_TASK_EXECUTION_PARAMS = "DELETE FROM %PREFIX%EXECUTION_PARAMS "
+			+ "WHERE task_execution_id in (:taskExecutionIds)";
+
+	private static final String DELETE_TASK_TASK_BATCH = "DELETE FROM %PREFIX%TASK_BATCH "
+			+ "WHERE task_execution_id in (:taskExecutionIds)";
+
+	@Override
+	public void deleteTaskExecutionsByTaskExecitionIds(Set<Long> taskExecitionIds) {
+		final MapSqlParameterSource queryParameters = new MapSqlParameterSource()
+				.addValue("taskExecutionIds", taskExecitionIds);
+		final String query = getQuery(DELETE_TASK_EXECUTIONS);
+		logger.info(">>>>>>>." + query);
+		this.jdbcTemplate.update(query, queryParameters);
+	}
+
+	@Override
+	public void deleteTaskExecutionParamsByTaskExecitionIds(Set<Long> taskExecitionIds) {
+		final MapSqlParameterSource queryParameters = new MapSqlParameterSource()
+				.addValue("taskExecutionIds", taskExecitionIds);
+		final String query = getQuery(DELETE_TASK_EXECUTION_PARAMS);
+		logger.info(">>>>>>>." + query);
+		this.jdbcTemplate.update(query, queryParameters);
+	}
+
+	@Override
+	public void deleteTaskTaskBatchRelationshipsByTaskExecitionIds(Set<Long> taskExecitionIds) {
+		final MapSqlParameterSource queryParameters = new MapSqlParameterSource()
+				.addValue("taskExecutionIds", taskExecitionIds);
+		final String query = getQuery(DELETE_TASK_TASK_BATCH);
+		logger.info(">>>>>>>." + query);
```

Comment:
update or remove logger info based on the notes above.

Reply:
Removed